functional-tests/build.gradle

```diff
@@ -0,0 +1,96 @@
+plugins {
+	id "groovy"
+	id "com.energizedwork.webdriver-binaries" version "1.4"
+}
+
+ext {
+	// todo add IE
+	drivers = ["remoteEdge", "remoteFirefox", "remoteChrome", "chrome"]
+
+	ext {
+		groovyVersion = '2.5.8'
```

Comment:
Make sure this matches the pod in the pipeline

Reply:
If you run into mystery failures in the pipeline run, it will likely be driven by version differences here

Reply:
Ah good point, I'll double check what the pod has.

Reply:
To be specific, this will typically manifest as headless driver failures in the logs.